app/routes/task_routes.py

```diff
@@ -1 +1,151 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from ..db import db
+from ..models.task import Task
+from .route_utilities import validate_model
+import json
+from datetime import datetime, timezone
+import requests
+import os
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.post("")
+def create_task():
```

Comment:
See my comment in `goal_routes.py` in `create_goal()` about how these two routes could be DRY'd up by writing a helper method.